src/client/pages/Offer/Introduction/Sidebar/StartDate.tsx

```diff
@@ -424,6 +462,31 @@ export const StartDate = ({
   )
 }
 
+type TooSoonSwitcherErrorModalProps = {
```

Comment:
Nit: this file is already huge, maybe we can split most of modal markup and styles into separate module?